data/json/overmap/overmap_special/specials.json

```
@@ -129,30 +129,32 @@
     "id": "Sewage Treatment Plant",
     "overmaps": [
       { "point": [ 0, -1, 0 ], "overmap": "road_end_north" },
-      { "point": [ 0, 0, 0 ], "overmap": "s_lot" },
-      { "point": [ 0, 1, 0 ], "overmap": "sewage_treatment" },
-      { "point": [ 0, 1, -1 ], "overmap": "sewage_treatment_hub" },
-      { "point": [ 0, 0, -1 ], "overmap": "sewage_treatment_under" },
-      { "point": [ -1, 0, -1 ], "overmap": "sewage_treatment_under" },
-      { "point": [ 1, 0, -1 ], "overmap": "sewage_treatment_under" },
-      { "point": [ -1, 1, -1 ], "overmap": "sewage_treatment_under" },
-      { "point": [ 1, 1, -1 ], "overmap": "sewage_treatment_under" },
-      { "point": [ -1, 2, -1 ], "overmap": "sewage_treatment_under" },
-      { "point": [ 0, 2, -1 ], "overmap": "sewage_treatment_under" },
-      { "point": [ 1, 2, -1 ], "overmap": "sewage_treatment_under" }
+      { "point": [ 0, 0, 0 ], "overmap": "sewage_treatment_0_0_0_north" },
+      { "point": [ 0, 0, 1 ], "overmap": "sewage_treatment_0_0_roof_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "sewage_treatment_1_0_0_north" },
+      { "point": [ 1, 0, 1 ], "overmap": "sewage_treatment_1_0_roof_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "sewage_treatment_0_1_0_north" },
+      { "point": [ 0, 1, 1 ], "overmap": "sewage_treatment_0_1_roof_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "sewage_treatment_1_1_0_north" },
+      { "point": [ 1, 1, 1 ], "overmap": "sewage_treatment_1_1_roof_north" },
+      { "point": [ 0, 0, -1 ], "overmap": "sewage_treatment_0_0_-1_north" },
+      { "point": [ 1, 0, -1 ], "overmap": "sewage_treatment_1_0_-1_north" },
+      { "point": [ 2, 0, -1 ], "overmap": "sewage_treatment_2_0_-1_north" },
+      { "point": [ 0, 1, -1 ], "overmap": "sewage_treatment_0_1_-1_north" },
+      { "point": [ 1, 1, -1 ], "overmap": "sewage_treatment_1_1_-1_north" },
+      { "point": [ 2, 1, -1 ], "overmap": "sewage_treatment_2_1_-1_north" }
     ],
+    "locations": [ "land" ],
     "connections": [
-      { "point": [ 0, -1, 0 ] },
-      { "point": [ 0, -1, -1 ], "terrain": "sewer", "connection": "sewer_tunnel", "from": [ 0, 0, -1 ] },
-      { "point": [ 2, 1, -1 ], "terrain": "sewer", "connection": "sewer_tunnel", "from": [ 1, 1, -1 ] },
-      { "point": [ 0, 3, -1 ], "terrain": "sewer", "connection": "sewer_tunnel", "from": [ 0, 2, -1 ] },
-      { "point": [ -2, 1, -1 ], "terrain": "sewer", "connection": "sewer_tunnel", "from": [ -1, 1, -1 ] }
+      { "point": [ 0, -1, 0 ], "terrain": "road" },
```

I can suggest adding an auto-connecting road by something like this:
```suggestion
      { "point": [ 0, -2, 0 ], "terrain": "road", "connection": "local_road", "from": [ 0, -1, 0 ] }
```